cpp/ql/src/Security/CWE/CWE-732/OpenCallMissingModeArgument.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name File opened with O_CREAT flag but without mode argument
+ * @description Opening a file with the O_CREAT flag but without mode argument reads arbitrary bytes from the stack.
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 7.8
+ * @precision medium
+ * @id cpp/open-call-with-mode-argument
+ * @tags security
+ *       external/cwe/cwe-732
```

Comment:
For this query I think we could upgrade the `severity` to `error`, because I can't see breaking this rule ever being a good idea.  For `precision`, we need to run the query on a fairly large sample of real world projects (either on lgtm.com or through the DCA job) and see what we get.

Reply:
I changed it to error.